packages/storybook/stories/components/drawer/drawer.stories.tsx

```diff
@@ -0,0 +1,225 @@
+import { Button } from '@ovhcloud/ods-react-button/src';
+import { DrawerOpenChangeDetail } from '@ovhcloud/ods-react-drawer/src/components/drawer/Drawer.tsx';
+import { type Meta, type StoryObj } from '@storybook/react';
+import React, { useState } from 'react';
+import {
+  Drawer,
+  DRAWER_POSITION,
+  DRAWER_POSITIONS,
+  DrawerBody,
+  DrawerContent,
+  DrawerContentProp,
+  type DrawerProp,
+  DrawerTrigger,
+} from '../../../../ods-react/src/components/drawer/src';
+import { CONTROL_CATEGORY } from '../../../src/constants/controls';
+import { excludeFromDemoControls, orderControls } from '../../../src/helpers/controls';
+import { staticSourceRenderConfig } from '../../../src/helpers/source.ts';
+
+type Story = StoryObj<DrawerProp>;
+type DemoArg = Partial<DrawerProp> & Partial<DrawerContentProp> & {
+  content?: string;
+}
+
+const meta: Meta<DrawerProp> = {
+  argTypes: excludeFromDemoControls(['defaultOpen', 'onOpenChange', 'open']),
+  component: Drawer,
+  subcomponents: { DrawerContent, DrawerTrigger },
+  title: 'ODS Components/Drawer',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  render: (arg: DemoArg) => (
+    <Drawer closeOnEscape={arg.closeOnEscape} closeOnInteractOutside={arg.closeOnInteractOutside}>
+      <DrawerTrigger asChild>
+        <Button>
+          Trigger Drawer
+        </Button>
+      </DrawerTrigger>
+
+      <DrawerContent position={arg.position}>
+        <DrawerBody>
+          { arg.content }
+        </DrawerBody>
+      </DrawerContent>
+    </Drawer>
+  ),
+  argTypes: orderControls({
+    closeOnEscape: {
+      table: {
+        category: CONTROL_CATEGORY.general,
+      },
+      control: { type: 'boolean' },
+    },
+    closeOnInteractOutside: {
+      table: {
+        category: CONTROL_CATEGORY.general,
+      },
+      control: { type: 'boolean' },
+    },
+    content: {
+      table: {
+        category: CONTROL_CATEGORY.slot,
+      },
+      control: 'text',
+    },
+    position: {
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'DrawerPosition.left' },
+        type: { summary: 'DrawerPosition' },
```

Comment:
would be nice to have the same control on every component using position, this differs from the rest